src/Browscap/Data/Factory/PlatformFactory.php

```diff
@@ -52,6 +52,12 @@ public function build(array $platformData, array $json, $platformName)
             );
         }
 
+        if (!array_key_exists('match', $platformData)) {
```

Comment:
FWIW, we should look into using `beberlei/assert` for stuff like this; simplifies these assertions